src/app/time-select/time-select.service.ts

```diff
@@ -0,0 +1,128 @@
+import {Injectable} from '@angular/core';
+import {Moment} from 'moment';
+import * as moment from 'moment';
+
+import {UtilsService} from '../common/services/utils/utils.service';
+import {ITimeSelectConfig} from './time-select-config.model';
+
+export type TimeUnit = 'hour' | 'minute' | 'second';
+export const FIRST_PM_HOUR = 12;
+
+@Injectable()
+export class TimeSelectService {
+  readonly DEFAULT_CONFIG: ITimeSelectConfig = {
+    hours12Format: 'hh',
+    hours24Format: 'HH',
+    meridiemFormat: 'A',
+    max: undefined,
+    maxTime: undefined,
+    min: undefined,
+    minTime: undefined,
+    minutesFormat: 'mm',
+    minutesInterval: 1,
+    secondsFormat: 'ss',
+    secondsInterval: 1,
+    showSeconds: false,
+    showTwentyFourHours: false,
+    timeSeparator: ':',
```

Comment:
I'm modeling the `DEFAULT_CONFIG` style off of `day-calendar.service` and `month-calendar.service`.  What's redundant?